Go/Go.sublime-syntax

```diff
@@ -416,45 +432,69 @@ contexts:
 
   match-floats:
     # Integer, no fraction, exponent
-    - match: \d+({{exponent}})\d+
-      scope: constant.numeric.float.go
-      captures:
-        1: punctuation.separator.exponent.go
+    - match: '{{ddigits}}{{exponent}}{{ddigits}}'
+      scope: constant.numeric.float.decimal.go
     # Integer, fraction, optional exponent
-    - match: \d+(\.)\d+(?:({{exponent}})\d+)?
-      scope: constant.numeric.float.go
+    - match: '{{ddigits}}(\.){{ddigits}}(?:{{exponent}}{{ddigits}})?'
+      scope: constant.numeric.float.decimal.go
       captures:
         1: punctuation.separator.decimal.go
-        2: punctuation.separator.exponent.go
     # Dot without fraction
     - match: \d+\.(?:{{exponent}}\d+)?
       scope: invalid.deprecated.go
     # Dot without integer
     - match: \.\d+(?:{{exponent}}\d+)?
       scope: invalid.deprecated.go
```

Comment:
As the `invalid.deprecated` scopes are not specific, They might be merged with the ones from line 471 and 427?

```
(?:\d+\.|\.\d+)(?:{{exponent}}\d+)?i?
```

... maybe in a dedicated context at the end as it would then match both imaginary and real floats?

Reply:
Resolved in e921238.